libusb/os/darwin_usb.c

```diff
@@ -553,11 +553,11 @@ static int darwin_device_from_service (struct libusb_context *ctx, io_service_t
   io_cf_plugin_ref_t *plugInInterface = NULL;
   IOReturn kresult;
   SInt32 score;
-  
+
   const int max_retries = 5;
 
   /* The IOCreatePlugInInterfaceForService function might consistently return
-     an "out of resources" error with certain USB devices the first time we run 
+     an "out of resources" error with certain USB devices the first time we run
      it. The reason is still unclear, but retrying fixes the problem */
```

Comment:
These whitespace changes seem unnecessary, but hopefully we can just run clang-format one day...